### ISSUE TITLE
More then a single camera

### ISSUE THREAD

**Issue by pablovela5620** [openxrlab/xrslam#26]
Considering most VR headsets use 2+ headsets (4 on quest 2/ quest pro/ pico 4/ etc...) are there any plans to be able to add more than just monocular visual-inertial slam?

**Comment by wangnancpp** [comment]
VR devices need to design a customized VIO algorithms for sensors. Currently, we have no plans to develop and release the VIO algorithm for head-mounted devices.

**Comment by pablovela5620** [comment]
> VR devices need to design a customized VIO algorithms for sensors. Currently, we have no plans to develop and release the VIO algorithm for head-mounted devices.

Makes sense, what about for stereo inertial systems like intel realsense T265 or D435i? Similar to how DroidSLAM allows for monocular/stereo/depth systems (without imu unlike this implementation). Is the target only monocular vio systems? Thanks for the response
